common/interpreter/wawaka_wasm/WasmCryptoExtensions.cpp

```diff
@@ -1062,3 +1063,49 @@ extern "C" bool parse_sgx_report_wrapper(
 
     return true;
 }
+
+/* ----------------------------------------------------------------- *
+ * NAME: _ecdsa_create_extended_key_wrapper(
+ * ----------------------------------------------------------------- */
+extern "C" bool ecdsa_create_signing_keys_from_extended_key_wrapper(
+    wasm_exec_env_t exec_env,
+    const int32 key_buffer_offset, // uint8_t*
+    const int32 key_buffer_length, // size_t
+    int32 private_buffer_pointer_offset, // char**
+    int32 private_length_pointer_offset, // size_t*
+    int32 public_buffer_pointer_offset,  // char**
+    int32 public_length_pointer_offset   // size_t*
+    )
+{
+    wasm_module_inst_t module_inst = wasm_runtime_get_module_inst(exec_env);
+    try {
+        pe::ThrowIf<pe::RuntimeError>(key_buffer_length != 48, "unsupported key size");
```

Comment:
Hmm, doesn't the length depend on the curve? So hardcoding 48 here seems a bit brittle?  (also, in principle the length could also be greater?)